control/src/CtBuffer.cpp

```diff
@@ -262,6 +262,13 @@ void CtBuffer::setup(CtControl *ct)
     hwNbBuffer = max_nbuffers;
   m_hw_buffer->setNbBuffers(hwNbBuffer);
 
+  int nbBuff;
+  m_hw_buffer->getNbBuffers(nbBuff);
+  if (nbBuff > 1)
+    m_hw_buffer->setNbBuffers(nbBuff);
+  else
+     m_hw_buffer->setNbBuffers(hwNbBuffer);
+
```

Comment:
I don't really understand what this is supposed to do because the number of buffer 
is always set at line 263. So nbBuff is always = to hwBuffer.....
What behaviour you need?

Reply:
OK this can be reverted